src/clib/lib/config/config_schema_item.cpp

```diff
@@ -388,8 +388,8 @@ bool config_schema_item_validate_set(const config_schema_item_type *item,
                     char *path = config_path_elm_alloc_abspath(path_elm, value);
                     if (!util_entry_exists(path)) {
                         error_list.push_back(util_alloc_sprintf(
-                            "Cannot find file or directory %s in %s ", value,
-                            config_path_elm_get_abspath(path_elm)));
+                            "Cannot find file or directory %s in path %s ",
```

Comment:
Only one more small comment as it seems you missed that part in my previous comment - i would also like to distinguish the file/dirname from the sentence as currently it becomes like this ( running the poly.ert and remove the observation file )
`Cannot find file or directory observations ... `

it would be much more clear that the file/dir defined wrong/not exists if the sentence is something like this
`Cannot find file or directory "observations" ... `
or 
`Cannot find file or directory <observations> ... `
or something else which makes sense.... Not sure if this is necessary - but i just feel that it is a bit more informative